PCF-ADR-0003-codeflare-release-process.md

```diff
@@ -0,0 +1,123 @@
+# Standardized release process for Project CodeFlare
+
+|                |                          |
+| -------------- | ------------------------ |
+| Date           | 04/28/2023               |
+| Scope          |                          |
+| Status         | Proposed                    |
+| Authors        | [Anish Asthana](@anishasthana) [Mustafa Eyceoz](@Maxusmusti) |
+| Supersedes     | N/A                      |
+| Superseded by: | N/A                      |
+| Issues         |                          |
+| Other docs:    | PCF-ADR-002              |
+
+## What
+
+This ADR introduces a unified release process, testing, and versioning scheme for Project CodeFlare.
+[PCF-ADR-002](./PCF-ADR-0002-release-story-and-branch-maintenance.md) is a related ADR to this one but is purely supplemental. ADR-002 covers the branching strategy to be used by all components, whereas this ADR is focused on higher
```

Comment:
Wouldn't these be lower level details as they are specific to the project?

Reply:
Discussed offline